src/effects/__mocks__/globals.js

```diff
@@ -1,2 +1,2 @@
 /* eslint-disable import/prefer-default-export */
-export const CURRENT_TERM = '4760';
+export const fetchCurrentTerm = () => {};
```

Comment:
This file can just be deleted, right?

Reply:
This is so I can run the reducer test for `globals` properly